InfiniTAM/ITMLib/Engines/Reconstruction/CPU/ITMSceneReconstructionEngine_CPU.tpp

```diff
@@ -128,7 +129,7 @@ void ITMSceneReconstructionEngine_CPU<TVoxel, ITMVoxelBlockHash>::AllocateSceneF
 	if (resetVisibleList) renderState_vh->noVisibleEntries = 0;
 
 	M_d = trackingState->pose_d->GetM(); M_d.inv(invM_d);
-
+    
```

Comment:
Please can you get rid of the whitespace changes?

Reply:
Yes, I can do it.